plutus-metatheory/src/Cost/Size.lagda.md

```diff
@@ -73,13 +74,13 @@ defaultConstantMeasure (tmCon (atomic aData) d) = dataSize d
 defaultConstantMeasure (tmCon (atomic aBls12-381-g1-element) x) = g1ElementSize x
 defaultConstantMeasure (tmCon (atomic aBls12-381-g2-element) x) = g2ElementSize x
 defaultConstantMeasure (tmCon (atomic aBls12-381-mlresult) x) = mlResultElementSize x
-defaultConstantMeasure (tmCon (list t) []) = 0
+defaultConstantMeasure (tmCon (list t) []) = 1
 defaultConstantMeasure (tmCon (list t) (x ∷ xs)) = 
-       defaultConstantMeasure (tmCon t x) 
-     + defaultConstantMeasure (tmCon (list t) xs)
+      3 + defaultConstantMeasure (tmCon t x)
```

Comment:
The Haskell version of this got changed in the Haskell code in #6285 so I've updated this to match.